libtransmission/verify.h

```diff
@@ -17,10 +17,18 @@
  * @{
  */
 
-void tr_verifyAdd(tr_torrent* tor, tr_verify_done_func callback_func, void* callback_user_data);
+typedef struct tr_verifier tr_verifier;
 
-void tr_verifyRemove(tr_torrent* tor);
+tr_verifier* tr_verifyInit(void);
 
-void tr_verifyClose(tr_session*);
+int tr_verifyGetParallelism(tr_verifier* v);
+
+void tr_verifySetParallelism(tr_verifier* v, int parallelism);
+
+void tr_verifyAdd(tr_verifier* v, tr_torrent* tor, tr_verify_done_func callback_func, void* callback_user_data);
+
+void tr_verifyRemove(tr_verifier* v, tr_torrent* tor);
```

Comment:
Not currently. I would suggest keeping it because this directly translates to moving to `class tr_verifier { void add(...); };` instead of infering this from the torrent object. WDYT?

(Note that after this change, all the tr_verify functions consistently take a tr_verifier argument.)